### PR TITLE
fix: add absolute path to default config

### PR DESCRIPTION
**Pull request by avivash**
## Status
**READY**

## Description
Adjusting the `output.path` in the `base.js`, so it doesn't have to modified locally for our projects. Also adding logic to add the production back to the config export automatically depending on the `process.env.NODE_ENV`.

## Impacted Areas in Application
List general components of the application that this PR will affect:

* `example/webpack.config.babel.js`
* `src/base.js`

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,12 +7,17 @@
 import { resolve } from 'path';
 import Config from 'webpack-config';
 
-export default new Config().extend(
+const config = new Config().extend(
```
Comment (alebiavati) [inline src/index.js]: Please revert all edits to this file as I already applied this change to the `production` config in #21

File: src/base.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ let config = new Config().merge({
   },
   output: {
     pathinfo: !inProduction,
-    path: 'build',
+    path: '/build',
```
Comment (alebiavati) [inline src/base.js]: Absolute path breaks webpack production build, but `webpack-dev-server` works fine.
Please use the following to support both:
```
{
  path: process.cwd() + '/build',
}
```

File: tests/acceptance/webpack.config.babel.js [hunk 3]
```diff
@@ -6,6 +6,10 @@
...
 
-const baseConfig = require('../../lib');
+import Config from 'webpack-config';
 
-module.exports = baseConfig;
+export default new Config().extend('../../../lib/index.js').merge({
+  output: {
+    path: 'build',
+  },
+});
```
Comment (alebiavati) [inline tests/acceptance/webpack.config.babel.js]: No need for this file to change if we use `process.cwd() + '/build',`

**Comment by avivash** [comment]
@alebiavati I've pushed changes based on your most recent comments

**Pull request merged**